eventdisplay/xml/FairXMLDetectorConf.h

```diff
@@ -0,0 +1,52 @@
+/********************************************************************************
+ * Copyright (C) 2020-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+/*
+ * FairXMLDetectorConf.h
+ *
+ *  Created on: 21 paź 2021
+ *      Author: Daniel Wielanek
+ *		E-mail: daniel.wielanek@gmail.com
+ *		Warsaw University of Technology, Faculty of Physics
+ */
+#ifndef CBMROOT_CORE_EVENTDISPLAY_XML_FAIRXMLDETECTORCONF_H_
+#define CBMROOT_CORE_EVENTDISPLAY_XML_FAIRXMLDETECTORCONF_H_
+
+#include "FairXMLEveConf.h"
+
+#include <Rtypes.h>
+#include <RtypesCore.h>
+#include <TString.h>
+#include <vector>
+
+class TGeoNode;
+
+class FairXMLDetectorConf : public FairXMLEveConf
+{
+    std::vector<FairXMLDetectorConf> fChild;
+    TString fName;
+    Int_t fRecursive;
+    Color_t fColor;
+    Char_t fTransparency;
+    Bool_t fDefColor;
+    Bool_t fDefTransparency;
+    void ChangeNode(TGeoNode* node) const;
+    void SetupSubNode(TGeoNode* node, Int_t depth) const;
+
+  public:
+    FairXMLDetectorConf(FairXMLNode* node = nullptr);
+    void SetupNode(TGeoNode* node);
+    void AddChild(FairXMLDetectorConf& set) { fChild.push_back(set); };
+    Int_t GetNChildren() const { return fChild.size(); }
+    TString GetDetectorName() const { return fName; }
+    const FairXMLDetectorConf& GetChild(Int_t n) { return fChild[n]; };
+    virtual ~FairXMLDetectorConf(){};
+    ClassDef(FairXMLDetectorConf, 1);
```

Comment:
Please use more override.

Are you expecting `FairXMLDetectorConf` to be used in Streamers? If not, please set the version to 0.